File: BeerViewer/Models/kcsapi/kcsapi_port.cs
namespace BeerViewer.Models.kcsapi
{
	public class kcsapi_port
	{
		public kcsapi_material[] api_material { get; set; }
		public kcsapi_deck[] api_deck_port { get; set; }
		public kcsapi_ndock[] api_ndock { get; set; }
		public kcsapi_ship2[] api_ship { get; set; }
		public kcsapi_basic api_basic { get; set; }
		public int api_combined_flag { get; set; }
		//public Api_Log[] api_log { get; set; }
	}
}

File: WindowFrame/src/KanColle/Raw/kcsapi_port.ts
// tslint:disable:class-name
import { kcsapi_material } from "@KC/Raw/kcsapi_material";
import { kcsapi_deck } from "@KC/Raw/kcsapi_deck";
import { kcsapi_ndock } from "@KC/Raw/kcsapi_dock";
import { kcsapi_basic } from "@KC/Raw/kcsapi_basic";
import { kcsapi_ship2 } from "@KC/Raw/kcsapi_ship";

export interface kcsapi_port {
	api_material: kcsapi_material[];
	api_deck_port: kcsapi_deck[];
	api_ndock: kcsapi_ndock[];
	api_ship: kcsapi_ship2[];
	api_basic: kcsapi_basic;
	api_combined_flag: number;
}
